### PR TITLE
keep more careful track of FFDCs encountered in FATs

### PR DESCRIPTION
**Pull request by wtlucy**

### CODE REVIEW

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
fixes #3943

File: dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java [hunk 1]
```diff
@@ -478,12 +528,14 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
...
-                                        oldInfo.count += entry.getValue().count;
+                                    // check to see if a set of FFDCInfo already existed
+                                    Set<FFDCInfo> oldInfo = ffdcPrimaryInfo.get(entry.getKey());
+                                    if (oldInfo == null) {
+                                        Set<FFDCInfo> set = new HashSet<FFDCInfo>();
+                                        set.add(entry.getValue());
+                                        ffdcPrimaryInfo.put(entry.getKey(), set);
                                     } else {
-                                        ffdcPrimaryInfo.put(entry.getKey(), entry.getValue());
+                                        ffdcPrimaryInfo.get(entry.getKey()).add(entry.getValue());
```
Comment (aguibert) [inline dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java]: If there is already a `Set<FFDCInfo>` for this key, we should check if there is a matching `FFDCInfo` in the set, and if so, increment the count, and if not do the current line 538 code

**Comment by wtlucy** [comment]
#build

**Pull request merged**